Ginger/GingerCoreNET/WorkSpaceLib/Solution.cs

```diff
@@ -227,6 +228,83 @@
 
         private string mAccount;
 
+        /// <summary>
+        /// For encrypting password variables
+        /// </summary>
+        public string EncryptionKey { get; set; }
+
+        [IsSerializedForLocalRepository]
+        public string EncryptedSolutionName {get;set;}
```

Comment:
Change the name of the field to EncryptedValidationString

Reply:
Changed to EncryptedValidationString